3696/1/scraper.rb

```diff
@@ -0,0 +1,39 @@
+module Scraper
+  FOR_USED_BY = '/network/dependents'.freeze
+  URL = 'https://rubygems.org/gems/'.freeze
+  # I can't think of better place for this method, because Monkey Patching is bad and,
+  # as I understand modules, they can be just a bunch of methods without dependency on the state
+  # :reek:UtilityFunction
+  def to_natural(num)
```

Comment:
Move into module and include the module into the Scrapper class (which does not exist yet >_<)